src/renderers/shaders/ShaderChunk/tonemapping_pars_fragment.glsl.js

```diff
@@ -105,12 +113,39 @@ vec3 agxDefaultContrastApprox( vec3 x ) {
 
 }
 
-// AgX Tone Mapping implementation based on Filament, which in turn is based
-// on Blender's implementation using rec 2020 primaries
-// https://github.com/google/filament/pull/7236
-// Inputs and outputs are encoded as Linear-sRGB.
+vec3 agxLook( vec3 color, int look ) {
 
-vec3 AgXToneMapping( vec3 color ) {
+    if ( look == AGX_LOOK_BASE ) {
+
+        return color;
+
+    }
+
+    const vec3 lw = vec3( 0.2126, 0.7152, 0.0722 );
+
+    float luma = dot( color, lw );
+
+    vec3 offset = vec3( 0.0 );
+    vec3 slope = vec3( 1.0 );
+    vec3 power = vec3( 1.0 );
+    float sat = 1.0;
+
+    if ( look == AGX_LOOK_PUNCHY ) {
+
+        slope = vec3( 1.0 );
+        power = vec3( 1.35, 1.35, 1.35 );
+        sat = 1.4;
+
+    }
```

Comment:
This can be simplified -- or are you leaving this for future look options?

Reply:
I was leaving space for future looks, but I don't particularly feel that we need to ship the "Golden" look out of the box, and I'm not sure if the contrast-oriented looks are implemented in the same way. So, I'd be happy to simplify this if you prefer!

Reply:
Up to you. :-)